themes/base.theme.bash

```diff
@@ -625,3 +625,14 @@ function _save-and-reload-history() {
 	local autosave=${1:-0}
 	[[ $autosave -eq 1 ]] && history -a && history -c && history -r
 }
+
+function venv_prompt() {
+	local python_venv=""
+	if [[ -n "${CONDA_DEFAULT_ENV:-}" ]]; then
+		python_venv="${CONDA_DEFAULT_ENV}"
+		PYTHON_VENV_CHAR=${CONDA_PYTHON_VENV_CHAR}
+	elif [[ -n "${VIRTUAL_ENV:-}" ]]; then
+		python_venv="${VIRTUAL_ENV##*/}"
+	fi
+	[[ -n "${python_venv}" ]] && echo "${PYTHON_VENV_CHAR}${python_venv}"
+}
```

Comment:
It has just come to my attention that we already have `virtualenv_prompt` and `condaenv_prompt`, so its not necessary to add an additional `venv_prompt`.

Reply:
The parenthesis would be added by the theme setting `VIRTUALENV_THEME_PROMPT_PREFIX='('` and `VIRTUALENV_THEME_PROMPT_SUFFIX=')'`.

Reply:
Thank you! I see, this gives more control over the theme. Didn't realize there are these options.

I've change the function to return either condaenv or venc depending on what exists. In this way if theme contributors want to add one function which would deal with either of two, they can do it like so. If not, the can use `virtualenv_prompt` or `condaenv_prompt` to customize even further. What do you think? :relaxed: